test/cutting/test_cutting_reconstruction.py

```diff
@@ -40,11 +45,6 @@ def test_cutting_reconstruction(self):
                 quasi_dists=[QuasiDistribution({"0": 1.0})], metadata=[{}]
             )
             weights = [(1.0, WeightType.EXACT)]
-            subexperiments = [QuantumCircuit(2)]
-            creg1 = ClassicalRegister(2, name="observable_measurements")
-            creg2 = ClassicalRegister(1, name="qpd_measurements")
-            subexperiments[0].add_register(creg1)
-            subexperiments[0].add_register(creg2)
```

Comment:
There was some unnecessary code in this file that I removed.  (`subexperiments` is constructed but then never used anywhere.)